app/src/main/java/de/rampro/activitydiary/ui/main/RedPacketActivity.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * ActivityDiary
+ *
+ * Copyright (C) 2024 Raphael Mack http://www.raphael-mack.de
```

Comment:
actually this file was invented by you, so why not place your name for copyright attribution? (At least additionally)

Reply:
I apologize for the PR, it's just our course homework, and I sent this PR just for the curiosity cause it's our first time to join a open source software collaboration... Really sorry for the interrupt.